src/inmanta/plugins.py

```diff
@@ -193,76 +191,26 @@ def clear(cls, inmanta_module: Optional[str] = None) -> None:
             cls.__functions = {}
 
 
-def report_missing_arguments(
-    func: str, missing_args: collections.abc.Sequence[str], args_sort: Literal["positional", "keyword-only"]
-) -> None:
+class Never(inmanta_type.Type):
```

Comment:
I would just make a Null type, this is very complicated 

I will also print itself as `Never?` in error messages I think.

Reply:
I renamed it to `Null`